feat(docs): Release Stage words (#3903)
getsentry/sentry-docs: src/docs/product/releases/health/index.mdx

The chart does not display if you don't select a project or if you select multiple projects.

### Adoption Stage

> Adoption stage is a high-level overview of the usage your release is seeing relative to other releases. There are three option stages: "High Adoption", "Low Adoption", and "Replaced". Every hour, we process your sessions over the last 6 hours and for each release (in a given environment and project), we'll calculate it's percentage of sessions across all sessions. If that percent is above a threshold (which is 10% divided by the number of releases in the last 6 hours), that release will be tagged as "High Adoption". If it was previously tagged as "High Adoption" but no longer meets the minimum threshold, it is labeled "Replaced". All other releases are marked as "Low Adoption". Note: "Replaced" releases that reach the threshold will once again be marked as "High Adoption".

Reviewer: ```suggestion
Adoption stage is a high-level overview of the usage your release is seeing relative to other releases. It is calculated every hour by processing your sessions over the last six hours. For each release (in a given environment and project), we calculate the release's percentage of sessions across all sessions into one of three option stages: 
- "High Adoption" - the release's percentage of sessions across all sessions is above a threshold (which is 10% divided by the number of releases in the last six hours) 
- "Low Adoption" - the release's percentage of sessions across all session that are below the threshold
- "Replaced" -  the release was previously tagged as "High Adoption", but no longer meets the minimum threshold
  
<Note>

"Replaced" releases that reach the threshold will once again be marked as "High Adoption".

</Note>
```

The chart does not display if you don't select a project or if you select multiple projects.

### Adoption Stage

> 

Reviewer 2: ```suggestion

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```

> Adoption stage is a high-level overview of the usage your release is seeing relative to other releases. It is calculated every hour by processing your sessions over the last six hours. For each release (in a given environment and project), we calculate the release's percentage of sessions across all sessions into one of three option stages:

Reviewer 2: ```suggestion
Adoption stage is a high-level overview of the usage your release is seeing relative to other releases. It is calculated every hour by processing your sessions over the last six hours. For each release, we calculate the release's percentage of sessions against all sessions in the same environment and project, and mark it as being in one of three stages:
```

- "High Adoption" - the release's percentage of sessions across all sessions is above a threshold (which is 10% divided by the number of releases in the last six hours)
- "Low Adoption" - the release's percentage of sessions across all session that are below the threshold
- "Replaced" -  the release was previously tagged as "High Adoption", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "High Adoption".

<Note>

> Adoption Stage UI elements are only displayed when filtered down to a single environment.

Reviewer: minor smoothing
```suggestion
Adoption Stage elements in sentry.io are displayed only when filtered to a single environment.
```
